src/controllers/wishes.controller.js

```diff
@@ -0,0 +1,37 @@
+import { wishListServices } from '../services';
+import { asyncWrapper } from '../helpers';
+
+const getWishList = asyncWrapper(async (req, res) => {
+  const wishlist = await wishListServices.getWishlistById(req.user.id);
+  return res.status(200).json({
+    code: 200,
+    message: 'retrieved all buyers wishlist.',
+    data: wishlist,
+  });
+});
+
+const addToWishlist = asyncWrapper(async (req, res) => {
+  const { pid } = req.params;
+  await wishListServices
+    .addProductToWishlist(req.user.id, pid)
+    .then((wishlist) =>
+      res.status(201).json({
+        code: 201,
+        message: 'Product Added To wishlist.',
+        data: wishlist,
+      })
+    );
+});
+
+const removeFromWishlist = asyncWrapper(async (req, res) => {
+  const { pid } = req.params;
+  await wishListServices
+    .removeProductFromWishlist(req.user.id, pid)
+    .then((wishlist) =>
```

Comment:
correct this. no need of `await` since you used `.then()`

Reply:
👍

Reply:
Solved the changes 👍